### PR TITLE
German navigation

### PR DESCRIPTION
**Pull request by motleydev**
**What**: Added German translations for navigation

**Why**: Because there was not German translation for the navigation.

**How**: I translated the navigation markdown files into German.

As best as I can tell, the default content will show through if there's no page content behind it?? This is not the way it works in development. In development the page urls simply break if I run the test with Lang=de

### CODE REVIEW

**Comment by kentcdodds** [comment]
Awesome! Thank you so much!!!

> In development the page urls simply break if I run the test with Lang=de

Run with `LOCALE=de`

**Comment by motleydev** [comment]
The translations work for the nav with the LOCALE (that's what I meant :)) but when I tried to navigate, the pages themselves weren't there
screen shot 2017-07-12 at 22 13 55

**Comment by kentcdodds** [comment]
Yeah, that error is probably because the footer yaml was invalid. I should probably improve error messages :smile:

File: components/content/de/nav.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+home: 'Home'
+gettingStarted: 'Anfangen'
+basics: 'Grundlagen'
+advanced: 'Fortschrittlich'
```
Comment (karsasmus) [inline components/content/de/nav.md]: change to 'Fortgeschritten'

File: components/content/de/nav.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+examples: 'Beispielen'
```
Comment (karsasmus) [inline components/content/de/nav.md]: change to 'Beispiele'

File: components/content/de/locale-chooser.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+help: 'Hilfen bei übersetzen'
+ariaLabelList: 'Sprachauswahl'
+ariaLabelButton: 'Wechsel Sprache'
```
Comment (karsasmus) [inline components/content/de/locale-chooser.md]: change to 'Sprache wechseln'

File: components/content/de/locale-chooser.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+help: 'Hilfen bei übersetzen'
```
Comment (karsasmus) [inline components/content/de/locale-chooser.md]: change to 'Hilf bei der Übersetzung'

**Comment by motleydev** [comment]
Thanks @karsasmus!

**Comment by kentcdodds** [comment]
@motleydev, do you mind adding @karsasmus to the contributors as a reviewer? Thanks so much @karsasmus!

**Comment by motleydev** [comment]
@kentcdodds Sure! How? If I scroll to the top it shows him listed under a reviewers section?

**Comment by kentcdodds** [comment]
Oh, I'm talking about [contributors on the repo](https://github.com/kentcdodds/glamorous-website/blob/master/README.md#contributors) ([instructions](https://github.com/kentcdodds/glamorous-website/blob/master/CONTRIBUTING.md#add-yourself-as-a-contributor)) :smile:

File: components/content/de/locale-chooser.md (hunk 4 above)
Comment (kairichard) [inline components/content/de/locale-chooser.md]: Although 'Hilf bei der Übersetzung' has an informal tone to it - "Bei der Übersetzung helfen" is a bit more formal #minor

**Comment by motleydev** [comment]
Aha. :) Was wondering what magical github feature you were referring to.

File: .all-contributorsrc [hunk 5]
```diff
@@ -346,6 +346,16 @@
       "contributions": [
         "example"
       ]
+    },
+    {
+      "login": "motleydev",
```
Comment (kentcdodds) [inline .all-contributorsrc]: Let's make sure to add @karsasmus and @kairichard to this as reviewers as well. Thanks!

File: components/content/de/nav.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+---
+home: 'Home'
+gettingStarted: 'Anfangen'
```
Comment (MoeSattler) [inline components/content/de/nav.md]: Maybe "Erste Schritte" oder "Einstieg" would be more appropriate.

Comment (motleydev) [inline components/content/de/nav.md]: I like this. Thanks!

File: .all-contributorsrc (hunk 5 above)
Comment (motleydev) [inline .all-contributorsrc]: Done! I just realized I was missing the actual "reviewer" property. Sorry!

**Comment by kentcdodds** [comment]
Wonderful! Thank you all very much!

**Pull request merged**